bigtable-dataflow-parent/bigtable-beam-import/src/test/java/com/google/cloud/bigtable/beam/hbasesnapshots/EndToEndIT.java

```diff
@@ -292,7 +292,7 @@ public void testHBaseSnapshotImport() throws Exception {
 
     // Validate the counters.
     Map<String, Long> counters = getCountMap(result);
-    Assert.assertEquals(counters.get("ranges_matched"), (Long) 101L);
+    Assert.assertEquals(counters.get("ranges_matched"), (Long) 100L);
```

Comment:
Where is the logic for this being changed? I don't see anything that should affect this counter in this PR.

Reply:
This was changed in https://github.com/googleapis/java-bigtable-hbase/pull/3586